src/AlertsManager.cpp

```diff
@@ -713,21 +739,59 @@ void AlertsManager::notifyAlert(AlertEntity alert_entity, const char *alert_enti
 
 /* **************************************************** */
 
+void AlertsManager::notifyToSlackIfNeeded (AlertEntity alert_entity, const char *alert_entity_value,
```

Comment:
notifyAlert and notifyToSlackIfNeeded do similar things and it is confusing to know when to call one or the other. Please refactor to make it more clear